seed/challenges/03-front-end-libraries/jquery.json

```diff
@@ -168,7 +168,8 @@
       ],
       "tests": [
         "assert($(\"button\").hasClass(\"animated\") && $(\"button\").hasClass(\"bounce\"), 'message: Use the jQuery <code>addClass&#40&#41</code> function to give the classes <code>animated</code> and <code>bounce</code> to your <code>button</code> elements.');",
-        "assert(!code.match(/class.*animated/g), 'message: Only use jQuery to add these colors to the element.');"
+        "assert(!code.match(/class.*animated/g), 'message: Only use jQuery to add these colors to the element.');",
+        "assert(code.match(/\\$\\(document\\)\\.ready\\(function.*(\\s|\\n)*.*button.*.addClass.*\\);/g), 'message: It is better practice to include all your jQuery code within <code>$(document).ready();</code>');"
```

Comment:
~It is better practice to include all your jQuery code within <code>$(document).ready();</code>~

Your jQuery code should be within the <code>$(document).ready();</code> function

Reply:
ok will do the changes